Networking/Networking/Remote/SitePostsRemote.swift

```diff
@@ -0,0 +1,46 @@
+import Foundation
+import Alamofire
+
+
+/// WP.com Site Posts API: Remote Endpoints
+///
+public class SitePostsRemote: Remote {
+
+
+    /// Loads the Post by ID from WP.com
+    ///
+    ///
+    /// - Parameters:
+    ///   - siteID: Site for which we'll fetch the post.
+    ///   - postID: Id of the post that we want to fetch
+    ///   - completion: Closure to be executed upon completion.
+    ///
+    public func loadSitePost(for siteID: Int64, postID: Int64, completion: @escaping (SitePost?, Error?) -> Void) {
+        let path = String(format: "/sites/%d/posts/%d", siteID, postID)
+        let request = DotcomRequest(wordpressApiVersion: .mark1_1, method: .get, path: path)
```

Comment:
do you think it's worth adding a `fields` param to only get the site ID and password? (I noticed the mock response `site-post.json` is quite long)